### PR TITLE
Please see descriptions on commits

### PR DESCRIPTION
**Pull request by pak-amsiq**
SwiftLint should be alright 👍

### CODE REVIEW

File: GBHFacebookImagePicker/Classes/Model/GBHFacebookPickerConfig.swift [hunk 1]
```diff
@@ -38,6 +38,12 @@ public struct GBHFacebookPickerConfig {
 
         /// Place checkView
         public var placeCheckView: CheckViewPosition = .bottomRight
+
+        /// Selected border color
+        public var selectedBorderColor: UIColor?
+
+        /// Selected border width
+        public var selectedBorderWidth: CGFloat = 0
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/Model/GBHFacebookPickerConfig.swift:46]: Can you please update the README / exemple with this two settings ? :-)

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift [hunk 2]
```diff
@@ -20,7 +20,8 @@ final class GBHSelectedView: UIView {
     // MARK: - Private var
     private var didSetupConstraints = false
 
-    private var checkMarkView: UIImageView?
+    private var checkMarkView: UIView?
+    private let checkMarkViewSize = CGSize(width: 20, height: 20)
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:24]: Maybe apply ratio size is preferable ? 
Like : 
`let size = self.frame.size.width / 5
CGSize(width:  size, height: size)` ? :)

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift [hunk 3]
```diff
@@ -8,6 +8,13 @@
 
 import UIKit
 
+public enum CheckViewPosition {
+    case topLeft
+    case topRight
+    case bottomLeft
+    case bottomRight
+}
+
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:17]: Cool feature :)

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift [hunk 4]
```diff
@@ -58,20 +65,17 @@ final class GBHSelectedView: UIView {
...
-                                              constant: -2.0))
+
+        // define margin
+        let margin: CGFloat = 2
+
+        // Place vertical constraint
+        self.applyVerticalConstraints(margin: margin)
+
+        // Place horisontal constraint
+        self.applyHorisontalConstraints(margin: margin)
```
Comment (floriangbh) [inline GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift]: horiZontale :-)

File: GBHFacebookImagePicker/Classes/Model/GBHFacebookPickerConfig.swift (hunk 1 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/Model/GBHFacebookPickerConfig.swift:46]: I think the README already include these settings: 

// Selected border color 
GBHFacebookImagePicker.pickerConfig.uiConfig.selectedBorderColor = UIColor.red

// Selected border width 
GBHFacebookImagePicker.pickerConfig.uiConfig.selectedBorderWidth = 4.0

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift (hunk 4 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift]: Fixed in commit

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift (hunk 3 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:17]: Thanks :)

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift (hunk 2 above)
Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:24]: The frame is zero at this moment.

**Comment by pak-amsiq** [comment]
I now replied to your feedback 👍

File: GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift (hunk 2 above)
Comment (floriangbh) [reply GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:24]: Yes, that's why you should initialize it in the init :) 
I will do it.

Comment (pak-amsiq) [reply GBHFacebookImagePicker/Classes/View/GBHSelectedView.swift:24]: Alright - Thanks! 👍

**Pull request merged**